rgw/v2/tests/s3_swift/test_user_bucket_rename.py

```diff
@@ -1,3 +1,16 @@
+"""test_user_bucket_rename - Test with Tenanted and Non-Tenanted User
+Usage: test_user_bucket_rename.py [options] <input>
+Options:
+    -c=<configs>    Configs [default: test_user_bucket_rename.yaml]
+Read configs and test_ops from <input> file. Use "-" for reading from stdin.
+
+Operation:
+    Create tenanted and non tenanted user
+    Create buckets for both the users
+    Rename buckets and users
+        Bucket unlink and link from non tenanted to tenanted users
+        Bucket unlink and link from tenanted to non tenanted users
+"""
 # test user and bucket rename with tenanted and non tenanted users
```

Comment:
I think this comment can be removed since we are adding more docstring information